.github/workflows/build_and_test.yml

```diff
@@ -128,14 +132,16 @@ jobs:
         path: wheelhouse
     - name: Install pip, wheel
       run: pip install -U pip wheel
+    - name: Install poetry
+      run: pip install poetry
     - name: Install extension
       run: for w in `find wheelhouse/ -type f -name "*.whl"` ; do pip install $w ; done
```

Comment:
I'm not sure what do do with this as I don't think its installed in the poetry env

Reply:
As above, I would replace this with: